plugin-packs/postcss-preset-env/README.md

```diff
@@ -354,6 +354,10 @@ postcssPresetEnv({
 });
 ```
 
+### debug
+
+The `debug` option enables debugging messages to stdout which should be useful to help you debug which features have been enabled/disabled and why.
```

Comment:
Should we use `result.warn` instead of `stdout`?
This can be done in `Once` or `OnceExit` and will use the plugin name as a prefix.

see : https://github.com/postcss/postcss/blob/b1bfc05bb70e414798af42ee9134d04c4d77285d/docs/guidelines/plugin.md#42-use-resultwarn-for-warnings

Reply:
Thought about that and there were two main reasons I didn't follow:

First is that this is not a warning but just debug data. 
Second is that it would mean queuing messages to be then spilled at once once it's done rather that as it goes which would be more useful if something odd happened?

Not a big lift though to change it. Let me know your thoughts!

Reply:
My main concern is that `stdout` should not be used for any output that isn't part of the main function of the program.

Piping output to a file or another program would become impossible if we add messages to `stdout`. (this would break our CLI for example)

Because of this I looked up the PostCSS API and guidelines and saw the notice to never use `console`. I am not sure which tools do not allow using `console`, and that also seems a bit weird to me.

I think the important bit is to not use `stdout` but maybe we need to ask PostCSS to clarify?

Reply:
@romainmenke think you're right. I've added a "simple" and naive approach. Let me know if you see any downsides.